incubator/autospotting/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
Should this match our default image build version?

Reply:
I figured I would go increment version numbers once the build of the container goes public and we have a latest version tag.